### DISCUSSION TITLE
Popup: can I set the color of line delimeters?

### DISCUSSION THREAD

**Discussion by vitaly-zdanevich** [neoclide/coc.nvim#3938] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/3514015/180220325-b3ba958d-b63b-47a6-abf6-10b423f6575e.png)

These two horizontal lines - I want to make them less bright.

**Comment by chemzqm** [comment]
You have to hack the code to add highlight groups at https://github.com/neoclide/coc.nvim/blob/2b2918a7d9e96f4e4c650edf5af6f4322391d300/src/markdown/index.ts#L71

**Comment by chemzqm** [comment, marked as answer]
Fixed on latest release branch of coc.nvim.

`:h CocFloatDividingLine`
